### DISCUSSION TITLE
UEFI Booting in Cosmos?

### DISCUSSION THREAD

**Discussion by AsertCreator** [CosmosOS/Cosmos#2367] [Ideas]
why? it will open smbios and nvram for cosmos (maybe) and make it load on machines afer 2012-2015. i tried implement uefi myself but i just broke cosmos build scheme.

**Comment by dm-vev** [comment]
Most likely not, as participants have noted in the problems before, they are trying to make the most flexible solution that would be applicable for older machines

However, everything is possible, but given that space has not yet been ported to arm, this is unlikely
